designcompose/src/androidTest/kotlin/com/android/designcompose/JniLiveWithTokenTests.kt

```diff
@@ -54,12 +53,12 @@ class JniLiveWithTokenTests {
     @Test
     fun invalidDocId() {
         assertFailsWith<FigmaFileNotFoundException> {
-            LiveUpdateJni.jniFetchDoc("InvalidDocID", firstFetchJson)
+            LiveUpdateJni.jniFetchDoc("InvalidDocID", firstFetchJson, ProxyConfig())
         }
     }
 
     private fun testFetch(docID: String) {
-        with(LiveUpdateJni.fetchDocBytes(docID, firstFetchJson)) {
+        with(LiveUpdateJni.fetchDocBytes(docID, firstFetchJson, ProxyConfig() = null)) {
```

Comment:
I'm not familiar with this syntax -- what does `ProxyConfig() = null` do?

Reply:
Possibly a paste error.  Fixed.